### PR TITLE
Enable Table Scroll

### LINKED ISSUE #11552: Security Vulnerabilities Table Not Fully Visible on Page
### Current State

(https://docs.meshery.io/project/security-vulnerabilities)

Screenshot 2024-08-05 010450

### Desired State

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by vaishnaviadhau**
**Notes for Reviewers**

This PR fixes #11552 

https://github.com/user-attachments/assets/66809e45-0d91-4336-a381-b26a80381d40

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 21e6d7a9f035bb57324af8851fd032b5a9d5578b at: https://66b26a58c900463e6eb60c5a--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit b66bfa3daec3404f4f767898057c622229919921 at: https://66b287a741b363622eb62e57--meshery-docs-preview.netlify.app

**Comment by vaishnaviadhau** [comment]
@yash37158, I have signed my commit.

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thanks! This amount of inline styling is overkill and redundant. Please refactor.

**Comment by Ashparshp** [comment]
@vaishnaviadhau let's discuss this PR during the Meshery dev meeting call, if you would. Please add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?usp=drivesdk).

**Comment by vaishnaviadhau** [comment]
> Thanks! This amount of inline styling is overkill and redundant. Please refactor.

@leecalcote I'll work on it

**Comment by vaishnaviadhau** [comment]
> @vaishnaviadhau let's discuss this PR during the Meshery dev meeting call, if you would. Please add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?usp=drivesdk).

@Ashparshp Okay

**Comment by l5io** [comment]
🚀 Preview for commit 6469397fe4465549c4c5456b662016e5e5486b20 at: https://66b37747b2f269007d787dcd--meshery-docs-preview.netlify.app

**Comment by sudhanshutech** [comment]
@vaishnaviadhau this the good item to discuss in website meeting today please add as agenda item for today: https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit and please jump to share your progress here at 5:30pm IST.

**Comment by Ashparshp** [comment]
@vaishnaviadhau, any updates?

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

File: docs/pages/project/security-vulnerabilities.md [hunk 1]
```diff
@@ -11,8 +11,27 @@ list: include
 
 ## List of Announced Vulnerabilities
 
-
-<table>
+<style>
```
[the remark this answers is not in the document]
Comment (vaishnaviadhau) [reply docs/pages/project/security-vulnerabilities.md:14]: @sudhanshutech okay I'll make the required changes

**Comment by l5io** [comment]
🚀 Preview for commit dc0e48d3388b7ce34a6f50b3960b93325b8acee1 at: https://66ccf264d759532162a36a61--meshery-docs-preview.netlify.app

**Review by sudhanshutech — APPROVED** [review]

**Pull request merged**